doc/tut2.rst

```diff
@@ -469,7 +469,8 @@ Generics
 ========
 
 Generics are Nim's means to parametrize procs, iterators or types
-with `type parameters`:idx:. They are most useful for efficient type safe
+with `type parameters`:idx:. Generic parameters are written within square
+brackets, for example ``Foo[T]``. They are most useful for efficient type safe
```

Comment:
`They are most useful for efficient type safe` weird grammar,
I would just link https://en.wikipedia.org/wiki/Generic_programming instead. :thinking:

Reply:
This file uses hard wrap, so that sentence continues on the next line. The full sentence:

> They are most useful for efficient type safe containers:

And the following paragraph is a Nim code example that implements an efficient type-safe container using generics. So I think the wording of that sentence is fine.

Also, note that I didn’t write the sentence we are discussing. The only change this commit makes is adding this sentence:

> Generic parameters are written within square brackets, for example ``Foo[T]``.

Reply:
You are right.